psscriptanalyzer/Initialize-Environment.ps1

```
@@ -0,0 +1,60 @@
+)
+
+$Global:EnvSettings = @{
+    Environment = $Environment
+    ConfigPath  = $ConfigPath
+    LoadedAt    = Get-Date
+}
+
+function Initialize-Vault {
+    param(
+        [string]$Username = "vault-admin",
+        [string]$Password = "V@ultM@ster2024"
```

PSScriptAnalyzer (1.25.0)

[error] Function 'Initialize-Vault' has both Username and Password parameters. Either set the type of the Password parameter to SecureString or replace the Username and Password parameters with a Credential parameter of type PSCredential. If using a Credential parameter in PowerShell 4.0 or earlier, please define a credential transformation attribute after the PSCredential type attribute.

(PSAvoidUsingUsernameAndPasswordParams)